intermediate-python-debugging/debugging-pytest.md

```diff
@@ -0,0 +1,55 @@
+# Debugging Tests With Pytest
+
+
+All the previous learnings on debugging apply to running and debugging code when running unit tests. However, VS Code has the ability to run individual tests from the editor and it allows you to run tests in the debugger.  This can help us identify why tests fail and get them to pass.
+
+## Goals
+
+We have practiced using a debugger tool while running Python programs. Our goal is to apply those same debugger skills when we are running unit tests.
+
+This lesson is structured like a follow-along exercise to illustrate:
```

Comment:
When you record this video, if you end up making scaffolding for this (aka you need some code with some tests), it might be worth it to somehow share that with the students.

I can imagine either of these options being fine:
- Make a repo and link it in this repo
- Insert the code snippets of the code + tests inside of this file, and ask the students to copy/paste them (just like how you did in the debugging in vs code lesson)